### PR TITLE
Added new tests for video

### PR DESCRIPTION
**Pull request by DrEuler**
Created new tests to check with valid return names and a valid cursor return value

### CODE REVIEW

File: tests/video/test_loading_cursor.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from panda3d.core import MovieVideo
+from panda3d.core import MovieVideoCursor
+import pytest
+
+def get_movie():
+    movie_path = "../samples/media-player/PandaSneezes.ogv"
+    getFile = MovieVideo.get(movie_path)
+    return getFile #Obtains a MovieVideo that references a file
+
+@pytest.fixture
```
Comment (Moguri) [inline tests/video/test_loading_cursor.py]: I doesn't really matter for one file, but fixtures should probably go in `tests/video/conftest.py`

File: tests/video/test_loading_cursor.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+def cursor_check():
+    movie_path = "../samples/media-player/PandaSneezes.ogv"
+    getFile = get_movie() #references the function above
+    valid_cursor_check = MovieVideo.open(getFile) #Open this video, returns a MovieVideoCursor 
+
+
+def test_loading():
+    getFile = get_movie()
+    FileReturn = MovieVideo.getFilename(getFile)
+    assert FileReturn == FileReturn
```
Comment (Moguri) [inline tests/video/test_loading_cursor.py]: A variable compared to itself should always be True, so this assert does nothing.

File: tests/video/test_loading_cursor.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+from panda3d.core import MovieVideoCursor
+import pytest
+
+def get_movie():
+    movie_path = "../samples/media-player/PandaSneezes.ogv"
+    getFile = MovieVideo.get(movie_path)
+    return getFile #Obtains a MovieVideo that references a file
+
+@pytest.fixture
+def cursor_check():
```
Comment (Moguri) [inline tests/video/test_loading_cursor.py]: This should `return` or `yield` a value to be usable as a fixture. As written, I am fairly certain it returns `None`.

File: tests/video/test_loading_cursor.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    movie_path = "../samples/media-player/PandaSneezes.ogv"
```
Comment (Moguri) [inline tests/video/test_loading_cursor.py]: `movie_path` is unused

File: tests/video/test_loading_cursor.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
+from panda3d.core import MovieVideo
+from panda3d.core import MovieVideoCursor
+import pytest
+
+def get_movie():
```
Comment (Moguri) [inline tests/video/test_loading_cursor.py]: This should be a fixture

File: tests/video/test_loading_cursor.py [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    valid_cursor_check = MovieVideo.open(getFile) #Open this video, returns a MovieVideoCursor 
+
+
+def test_loading():
+    getFile = get_movie()
+    FileReturn = MovieVideo.getFilename(getFile)
+    assert FileReturn == FileReturn
+
+def test_cursor_check(cursor_check):
+    valid_cursor_check = cursor_check
```
Comment (Moguri) [inline tests/video/test_loading_cursor.py]: This does nothing

File: tests/video/test_loading_cursor.py (hunk 2 above)
Comment (Moguri) [reply tests/video/test_loading_cursor.py]: I think you might want to do `assert FileReturn.open() is not None`

File: tests/video/test_loading_cursor.py [hunk 7]
```diff
@@ -0,0 +1,9 @@
+from panda3d.core import MovieVideo
+import pytest
+
+
+def test_cursor_check(): 
+    movie_path = "../../samples/media-player/PandaSneezes.ogv"
+    get_File = MovieVideo.get(movie_path)
```
Comment (Moguri) [inline tests/video/test_loading_cursor.py]: To follow PEP8, this should be `get_file`

File: tests/video/test_loading_cursor.py [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+    cursor_type = MovieVideo.open(get_File)
+    assert cursor_type != None
```
Comment (Moguri) [inline tests/video/test_loading_cursor.py]: Use `is not None` instead of `!= None` https://stackoverflow.com/questions/2209755/python-operation-vs-is-not

**Comment by DrEuler** [comment]
Changes for this test are also in #612 along with other new tests

**Pull request closed without merging**